Wikipedia/UI-V5/WMFRelatedSectionController.m

```diff
@@ -108,6 +122,21 @@ - (void)configureCell:(UICollectionViewCell*)cell
     }
 }
 
+- (UIViewController*)moreViewController {
+    #warning Remove SessionSingleton
+    WMFRelatedTitleListDataSource* dataSource =
```

Comment:
is the session singleton necessary? The related section controller already has access to the user data store right? And the user data store has the datastore as a property…